src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs

```diff
@@ -56,8 +56,10 @@ public App()
 		/// <param name="e">Details about the launch request and process.</param>
 		protected override void OnLaunched(LaunchActivatedEventArgs e)
 		{
-			Uno.Material.Resources.Init(this, new ResourceDictionary() { Source = new Uri("ms-appx:///ColorPaletteOverride.xaml") });
```

Comment:
Is there any particular reason as to why this line is expanded?

Reply:
Just made it match what we are doing in Gallery, for some reason otherwise the colors weren't being properly overridden

Reply:
Can you try to restore the old `Init` method, and replace the 2nd argument by a func?:
```cs
public static void Init(Application app, Func<ResourceDictionary> colorPaletteOverride = null)
{
//...
if (colorPaletteOverride?.Invoke() is ResourceDictionary override)
{
     app.Resources.MergedDictionaries.Add(override);
}
//...
}
```
And, see if this works? My suspicion is that the order at which the res-dicts are initialized played a role here.

Reply:
Updated and seems to work

Reply:
@MatFillion The mystery was finally solved!

Reply:
Hmm interesting! Good find! You can look maybe we have an open issue we could close🚀🚀

Reply:
Hit that approve button :)

Reply:
> ... And, be sure to subscribe to the repository and leave a comment below. XD

Jokes asides, I checked github and azure devops and there seems no open issue/work-item regarding this.